client/repodlgs_common.cpp

```diff
@@ -150,6 +150,63 @@ void get_economy_report_units_data(struct unit_entry *entries,
   unit_type_iterate_end;
 }
 
+/**
+   Returns an array of units data. Number of units in
+   the array is added to num_entries_used.
```

Comment:
New code should have the *'s
```suggestion
 * Returns an array of units data. Number of units in
 * the array is added to num_entries_used.
```